csrankings.csv

```diff
@@ -1924,8 +1923,8 @@ Cameron Browne,Maastricht University,http://cambolbro.com,e6dyku8AAAAJ
 Camillo J. Taylor,University of Pennsylvania,http://www.cis.upenn.edu/~cjtaylor,vMDCJHcAAAAJ
 Campbell Wilson,Monash University,http://monash.edu/research/explore/en/persons/campbell-wilson(3e5445ab-5a7f-43c9-bb99-6fde0a2d0962).html,NOSCHOLARPAGE
 Can Alkan,Bilkent University,http://www.cs.bilkent.edu.tr/~calkan,wo5_FngAAAAJ
-Can C. Özturan,Boğaziçi University,https://www.cmpe.boun.edu.tr/~ozturan,--uJKY0AAAAJ
-Can Özturan,Boğaziçi University,https://www.cmpe.boun.edu.tr/~ozturan,--uJKY0AAAAJ
+Can C. Özturan,Boğaziçi University,https://www.cmpe.boun.edu.tr/~ozturan,#NAME?

```

Comment:
These entries are artifacts of using Excel to edit the .csv file. It doesn't work. Please try again.